src/containers/BookList.js

```diff
@@ -2,27 +2,50 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
+import { removeBook } from '../actions/index';
 
 const mapStateToProps = state => ({ books: state.books });
+const select = dispatch => ({ removeBook: book => dispatch(removeBook(book)) });
 
-const ConnectedBookList = ({ books }) => (
-  <>
-    <table>
-      {
-        books.map(item => (
-          <tbody key={item.title}>
-            <Book books={item} />
-          </tbody>
-        ))
-      }
-    </table>
-  </>
-);
+const ConnectedBookList = ({ books, removeBook }) => {
+  // const { id, title } = books;
```

Comment:
## Remove commented out code
- [ ] The entire codebase looks professional saved for this part where we have commented code. It does make it appear untidy. I would suggest if there be a need we always rely on git history if we want to come back to something later on. Do you mind removing this piece of commented code?